save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -432,6 +499,11 @@ class TestSuitesSourceController(
                     "TestSuitesSource not found by name $name for organization $organizationName"
                 }
 
+    private fun getTestSuitesSource(id: Long): Mono<TestSuitesSource> =
+            blockingToMono {
+                testSuitesSourceService.findById(id)
+            }
```

Comment:
Maybe process situation when source is not found by id here?

Reply:
yes, sure -- will add